### PR TITLE
GA added to website

### PR DESCRIPTION
**Pull request by mwernimont**
Added both our and USGS GA scripts to application. @isuftin mind verifying I didn't screw it up?

### CODE REVIEW

File: nar_common/templates/base.html [hunk 1]
```diff
@@ -46,6 +46,16 @@
...
 		<script type="text/javascript" src="{% static 'nar_ui/js_lib/select2.js' %}"></script>
 		<script type="text/javascript" src="{% static 'nar_ui/util.js' %}"></script>
+		<script>
+			(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function()
+			{ (i[r].q=i[r].q||[]).push(arguments)}
+			,i[r].l=1*new Date();a=s.createElement(o),
+			m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+			})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+			ga('create', 'UA-58841817-1', 'auto');
+			ga('send', 'pageview');
```
Comment (isuftin) [inline nar_common/templates/base.html]: You need to add ga('set', 'anonymizeIp', true);

See https://github.com/isuftin/coastal-hazards/blob/master/coastal-hazards-portal/src/main/webapp/WEB-INF/jsp/components/common/google-analytics.jsp

**Comment by isuftin** [comment]
http://communities.usgs.gov/blogs/usgsdev/third-party-tools/google-analytics/google-analytics-instructions/

http://communities.usgs.gov/blogs/usgsdev/third-party-tools/google-analytics/google-analytics-faqs/

**Comment by mwernimont** [comment]
@isuftin Es bueno?

**Comment by isuftin** [comment]
:+1:

**Pull request merged**